File: rand.js
#!/usr/bin/node
function random(m, a, c, x){
  return (a * x + c) % m;
}

var args = process.argv

if(args.length == 6){
  let [i, j, m, a, c, x] = args;
  console.log(random(parseInt(m),parseInt(a),parseInt(c),parseInt(x)));
}
else {
  console.log("Error: Not matching argument count.");
}

File: rand.java
public class rand {
  public static int random(int m, int a, int c, int x){
    return (a * x + c) % m;
  }

  public static void main(String[] args) {
    if (args.length == 4)
      System.out.print(random(Integer.parseInt(args[0]), Integer.parseInt(args[1]), Integer.parseInt(args[2]), Integer.parseInt(args[3])));
    else
      System.out.println("Error: Not matching argument count.");
  }
}
